bundle/Form/FieldTypeHandler/Country.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\InformationCollectionBundle\Form\FieldTypeHandler;
+
+use Ibexa\Contracts\Core\FieldType\Value;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Ibexa\Contracts\Core\Repository\Values\ContentType\FieldDefinition;
+use Ibexa\Core\FieldType\Country\Value as CountryValue;
+use Netgen\Bundle\InformationCollectionBundle\Form\FieldTypeHandler;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use function array_flip;
+use function array_key_exists;
+use function array_keys;
+use function is_array;
+use function reset;
+
+final class Country extends FieldTypeHandler
+{
+    /**
+     * Country codes.
+     */
+    protected array $countryData;
+
+    /**
+     * Removed redundant data from array.
+     */
+    protected array $filteredCountryData;
+
+    public function __construct(array $countryData)
+    {
+        $this->countryData = $countryData;
+
+        foreach ($countryData as $countryCode => $country) {
+            $this->filteredCountryData[$countryCode] = $country['Name'];
+        }
+    }
+
+    public function convertFieldValueToForm(Value $value, ?FieldDefinition $fieldDefinition = null)
```

Comment:
Also inline-hint Country Value (and in other handlers as well).

Reply:
solved in fac3dfd3